example/generated-src/objc/TextSort-Bridging-Header.h

```diff
@@ -1,6 +1,17 @@
 // AUTOGENERATED FILE - DO NOT MODIFY!
 // This file generated by Djinni
 
+// TextSort_Bridging_Header.h
+// TextSort_Bridging_Header
+
+#import <Foundation/Foundation.h>
+
+//! Project version number for TextSort_Bridging_Header.
+FOUNDATION_EXPORT double TextSort_Bridging_HeaderVersionNumber;
```

Comment:
I'd prefer to have a `_` separating `TextSort_Bridging_Header` and `VersionNumber` so that it's `TextSort_Bridging_Header_VersionNumber`.

Reply:
I choosed the Xcode Convention. By calling a framework `MyLib`, the two variables are named `MyLibVersionNumber` and `MyLibVersionString`

Reply:
I see, thanks for the explanation.
In that case, would it make sense to name it `TextSortBridgingHeaderVersionNumber`?

Reply:
I guess yes 🤔